server/portal/apps/projects/models/utils.py

```diff
@@ -32,7 +32,7 @@ def get_latest_project_storage(max_project_id=None):
         )
         if '-' not in prj_id:
             continue
-        _, prj_id = prj_id.rsplit('-')
+        prj_id = prj_id.rsplit('-')[-1]
```

Comment:
should the same logic get updated in `get_latest_project_directory` (line 56) ?

Reply:
I think that line should be okay. since it's using `rsplit('-', 1), it will split the string into two parts, along the last delimiter (`-`)

Reply:
cool. 👍